cmd/cli/cmd/alpha/alpha.go

```diff
@@ -15,7 +15,7 @@ func NewCmd() *cobra.Command {
 	}
 
 	cmd.AddCommand(
-		manifestgen.NewCmd(),
+		generate.NewCmd(),
```

Comment:
it should be removed, as we move it out of `alpha`. Do not forget to regenerated docs